src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String inputLine = "";
+            while ((inputLine = bufferedReader.readLine()) != null) {
+                stringBuilder.append(inputLine.toLowerCase().replaceAll("[^a-z]", ""));
+            }
+        }
+
+        TreeSet<Character> stringSet = new TreeSet<>();
+        for (int i = 0; i < stringBuilder.length(); i++) {
+            stringSet.add(stringBuilder.charAt(i));
+        }
+
+        stringBuilder.setLength(0);
+        int resultSize = Math.min(stringSet.size(), 5);
```

Comment:
move this magic number to constant field

Reply:
Renamed to NAME_LENGTH.